sound/soc/sof/sof-priv.h

```diff
@@ -371,6 +371,10 @@ struct snd_sof_dev {
 	 */
 	struct snd_soc_component_driver plat_drv;
 
+	/* power status related */
+	u32 pcm_active;		/* Count of active PCM substream */
+	u32 lp_pcm_active;	/* Count of active low power PCM substream */
```

Comment:
no need for fixed-width, just use unsigned int.

Reply:
make sense.